commit 707200f3d046d097c47f73b1cfe14fa8d2519d32 (refs/changes/05/1305/3)
Author: Richard Kiene <richard.kiene@joyent.com>
Date:   2017-01-20T23:24:17+00:00 (2 years, 9 months ago)
    
    CMON-37 Add 'cmon-agent' to the shar
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/agents.json b/agents.json
index 7aa0cd9..ef5f68e 100644
--- a/agents.json
+++ b/agents.json
@@ -1,5 +1,6 @@
 [
   "agents_core/agents_core",
+  "cmon-agent/cmon-agent",
   "cn-agent/cn-agent",
   "net-agent/net-agent",
   "vm-agent/vm-agent",
diff --git a/package.json b/package.json
index a44c1d5..7000262 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "agents-installer",
   "description": "Build agentsshar package for SDC",
-  "version": "2.0.0",
+  "version": "2.1.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
